config/feed-variants/20220415-incumbent.json

```diff
@@ -0,0 +1,100 @@
+{
```

Comment:
[Suggestion]: It looks like there's some data being conveyed in this file name. Consider a memo or description comment or field inside this json file!

Reply:
My original thoughts were to include the name in the file.  However, I'm going with "convention over configuration".  I think a README.md in the config/feed-variants might suffice.